indico/modules/auth/templates/reset_password.html

```diff
@@ -21,6 +21,7 @@
 
         {{ form_header(form) }}
         {{ form_rows(form, widget_attrs=widget_attrs) }}
+        {{ template_hook('below-account-form') }}
```

Comment:
You use this in many different places, with no context whatsoever. I don't think this is a ghood idea. Either pass an argument indicating where it is, but maybe separate hooks (`reset-password-below-account-form` etc.) may be better.